Lib/test/test_logging.py

```diff
@@ -3658,6 +3676,10 @@ def test_bad_format_with_dollar_style(self):
         self.assertEqual(sorted(logging.getHandlerNames()),
                          ['bufferGlobal', 'fileGlobal'])
 
+    def test_invalid_key(self):
+        with self.assertRaises(ValueError):
```

Comment:
Can you check the error message as well?

I would prefer to declare config_invalid_key here, instead of 400 lines from here.

Reply:
I can do that, but there's an unsolved discussion in the issue #103008 and I don't know if we are pushing this forward.

Reply:
Check the current error message. It can be updated later.